### ISSUE TITLE
Does this support Legacy BIOS?

### ISSUE THREAD

**Issue by vsaucing** [Alex313031/ThoriumOS#78]
i dont have UEFI so i want to run this on legacy BIOS. does this have support for BIOS?

**Comment by TheSystemGuy1337** [comment]
> i dont have UEFI so i want to run this on legacy BIOS. does this have support for BIOS?

I don't think so. ChromeOS uses a proprietary bootstrap ROM variant of UEFI, so most likely ThoriumOS also requires UEFI. Also, this project is abandoned.

**Comment by tech128** [comment]
It absolutely has support for BIOS.  I was able to load it in qemu without efi support and it also loads on my HP Pavilion dv6 which is BIOS only as well.

As for the abandoned part, well it just had an update a little more than a month ago.  I'd hardly call that abandoned.

**Comment by Alex313031** [comment]
@TheSystemGuy1337 @tech128 @vsaucing Yes, it does support BIOS.

@TheSystemGuy1337 ChromeOS does, to integrate with the special firmware Chromebooks use. ChromiumOS and derivatives have support for all three: BIOS, UEFI, and chromebook firmware.

chromebook firmware is based on coreboot (which is why Mr.Chromebox firmware works), plus proprietary additions by google.
Also, its not abandoned.
